### PR TITLE
updating `Manta Network` parachain ID

### PR DESCRIPTION
**Pull request by zhenfeizhang**

### CODE REVIEW

File: packages/apps-config/src/endpoints/testingRococo.ts [hunk 1]
```diff
@@ -263,7 +263,7 @@ export function createRococo (t: TFunction): LinkOption[] {
         // Needs paraId >= 1000, below that is reserved for system parachains
         // Update paraId to new selection, remove this flag to enable
         isDisabled: true,
-        paraId: 777,
+        paraId: 7777,
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testingRococo.ts:266]: Just remove the isSisabled above the paraId as well. (Otherwise it does not show up)

**Review by jacogr — COMMENTED** [review]
One small comment.

File: packages/apps-config/src/api/spec/manta.ts [hunk 2]
```diff
@@ -11,7 +11,21 @@ const definitions: OverrideBundleDefinition = {
...
-      types: {}
+      types: {
+        Address: 'AccountId',
+        LookupSource: 'AccountId',
+        CurrencyId: {
+          "_enum": [
+            "Native",
+            "DOT",
+            "KSM",
+            "BTC"
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/manta.ts:22]: For these, just run `yarn lint` locally and it will be auto-fixed.

Comment (zhenfeizhang) [reply packages/apps-config/src/api/spec/manta.ts]: Thanks! Fixed.

File: packages/apps-config/src/endpoints/testingRococo.ts (hunk 1 above)
Comment (zhenfeizhang) [reply packages/apps-config/src/endpoints/testingRococo.ts]: Thanks! Fixed.

**Review by jacogr — APPROVED** [review]

**Pull request merged**